CONTRIBUTING.md

```diff
@@ -89,3 +89,6 @@ Output format of any non-plumbing command can change in non-backward-compatible
 1. Thanks to the courtesy of [Ila&iuml; Deutel](https://github.com/ilai-deutel),
    a [git-machete package](https://aur.archlinux.org/packages/git-machete) is hosted in Arch User Repository (AUR).
    If the release introduces significant changes/critical bugfixes, please [flag the package as out of date](https://aur.archlinux.org/pkgbase/git-machete/flag).
+
+1. Verify if changes you made in [blog-1.md](https://github.com/VirtusLab/git-machete/blob/develop/docs/blog-1/blog-1.md) and [blog-2.md](https://github.com/VirtusLab/git-machete/blob/develop/docs/blog-2/blog-2.md) are reflected in corresponding medium articles: [Make your way through the git (rebase) jungle with Git Machete](https://medium.com/virtuslab/make-your-way-through-the-git-rebase-jungle-with-git-machete-e2ed4dbacd02) or [Git Machete Strikes again!](https://medium.com/virtuslab/git-machete-strikes-again-traverse-the-git-rebase-jungle-even-faster-with-v2-0-f43ebaf8abb0).
```

Comment:
Whoops... CI failed on this one, as expected :/

Reply:
The problem is that you link to `docs/blog-1/blog-1.md` on `develop` branch... but such file does NOT exist yet on `develop` :/

Reply:
That's why you need to split the action into **multiple steps**:
1. merge this PR #145 while still referring to `copy-docs` branch in the links
2. open another PR that changes the branch in links from `copy-docs` to `develop`
3. merge that another PR
4. remove `copy-docs` branch